generate/scripts/generateJson.js

```diff
@@ -222,7 +222,7 @@ module.exports = function generateJson() {
     _.merge(enumerable, _.omit(override, ["values"]));
 
     output.push(enumerable);
-  }).value();
+  });
```

Comment:
don't we need the `.value` to make lodash actually do the operation? I thought they were lazy about that as of v4?